api/src/main/java/org/openmrs/GlobalProperty.java

```diff
@@ -15,7 +15,6 @@
 import org.openmrs.customdatatype.CustomDatatype;
 import org.openmrs.customdatatype.CustomDatatypeUtil;
 import org.openmrs.customdatatype.CustomValueDescriptor;
-import org.openmrs.customdatatype.InvalidCustomValueException;
```

Comment:
Is this the only unused import in the entire codebase?

Reply:
@f4ww4z according to sonar yeah only one is there

Reply:
I said in the entire codebase. Please merge this PR with #3107 then close this.